config.json

```diff
@@ -27,6 +27,10 @@
 			"method": "ebsi",
 			"url": "http://driver-did-ebsi:9080/",
 			"propertiesEndpoint": "true"
+		}, {
+			"method": "oyd",
+			"url": "https://oydid-registrar.data-container.net/",
```

Comment:
Shouldn't this point to the local Docker container (e.g. `http://driver-did-oyd:3000/`) instead of a remote URL?

Reply:
OK - updated to local Docker instance
(Uniresolver uses the remote instance hosted by OwnYourData and therefore, I thought it will be the same for Uniregistrar)